### DISCUSSION TITLE
Upstream Syncthing

### DISCUSSION THREAD

**Discussion by mxsrm** [MobiusSync/MobiusSync#133] [General]
Syncthing is now at v1.26.1 upstream. Is there an update planned in the future to bring Möbius up to speed?

**Comment by egigoka** [comment]
No commits in core for 8 months. I think it's ded :(

**Reply by mxsrm** [reply]
I switched to Resilio. Case closed.

**Reply by egigoka** [reply]
I mean, Resilio app haven't updated in two years on iOS lol

**Comment by SimonPickup** [comment]
See https://github.com/MobiusSync/MobiusSync/discussions/156
